services/py-api/src/utils.py

```diff
@@ -43,3 +49,26 @@ async def stop(self) -> None:
             raise RuntimeError("The AsyncClient has not been initialized!")
 
         await self._async_client.aclose()
+
+class JwtUtility:
+ 
+    @staticmethod
+    def encode_data(data: Participant) -> str:
+        return encode(data,key=environ["SECRET_KEY"])
+    
+    @staticmethod
+    def decode_data(token: str) -> Result[JwtUserData, str]: 
+     try: 
+        return Ok(JwtUserData(**decode(token, environ["SECRET_KEY"])))
+     except ExpiredSignatureError:
+            LOG.warning("Failed to decode the JWT token: The token has expired.")
+            return Err("The jwt token has expired.")
+     except InvalidSignatureError:
+            LOG.warning("Failed to decode the JWT token: The token has an invalid signature.")
+            return Err("The jwt token is invalid.")
+     except DecodeError:
+            LOG.warning("Failed to decode the JWT token: There was an error decoding the token.")
+            return Err("There was an error decoding the token.")
+    
+a= JwtUtility.decode_data("decoding data...")
+a.ok_value
```

Comment:
Remove this.

Reply:
Fixed in be223527926be992bf6729e80ee043728b8fa1c5